tutorials/smart-contracts/deploy-nft.md

```diff
@@ -0,0 +1,131 @@
+
+## Introduction
+
+Non-Fungible Tokens (NFTs) represent unique digital assets commonly used for digital art, collectibles, gaming, and identity verification. Asset Hub supports EVM-compatible smart contracts through PolkaVM, enabling straightforward NFT deployment. This tutorial guides you through deploying an [ERC721](https://eips.ethereum.org/EIPS/eip-721){target=\_blank} NFT contract on the Westend TestNet using the [Polkadot Remix IDE](https://remix.polkadot.io){target=\_blank}, a web-based development environment. This tutorial uses [OpenZeppelin's NFT contracts]({{ dependencies.open_zeppelin_contracts.repository_url}}/tree/{{ dependencies.open_zeppelin_contracts.version}}){target=\_blank} implementation to ensure security and standard compliance.
+
+## Prerequisites
+
+Before starting, make sure you have:
+
+- MetaMask installed and connected to Westend Asset Hub
+- A funded account with some WND tokens (you can get them from the [Westend Faucet](https://faucet.polkadot.io/westend?parachain=1000){target=\_blank})
+- Basic understanding of Solidity and NFTs
+
+## Creating the NFT Contract
+
+To create the NFT contract, you can follow the steps below:
+
+1. Navigate to the [Polkadot Remix IDE](https://remix.polkadot.io/){target=\_blank}
+2. Click in the **Create new file** button under the **contracts** folder, and name your contract as `MyNFT.sol`
+
+    ![](/images/tutorials/smart-contracts/deploy-nft/deploy-nft-1.webp)
+
+3. Now, paste the following NFT contract code into the editor
+
+    ```solidity title="MyNFT.sol"
+    --8<-- 'code/tutorials/smart-contracts/deploy-nft/MyNFT.sol'
+    ```
+
+    The key components of the code above are:
+
+    - Contract Imports
+
+        - [`ERC721.sol`]({{ dependencies.open_zeppelin_contracts.repository_url }}/blob/{{ dependencies.open_zeppelin_contracts.version }}/contracts/token/ERC721/ERC721.sol){target=\_blank} - the base contract for non-fungible tokens, implementing core NFT functionality like transfers and approvals
+        - [`Ownable.sol`]({{ dependencies.open_zeppelin_contracts.repository_url }}/blob/{{ dependencies.open_zeppelin_contracts.version }}/contracts/access/Ownable.sol){target=\_blank} - provides basic authorization control, ensuring only the contract owner can mint new tokens
+    
+    - Constructor Parameters
+
+        - `initialOwner` - sets the address that will have administrative rights over the contract
+        - `"MyToken"` - the full name of your NFT collection
+        - `"MTK"`- the symbol representing your token in wallets and marketplaces
+
+    - Key Function
+
+        - [`_safeMint(to, tokenId)`]({{ dependencies.open_zeppelin_contracts.repository_url }}/blob/{{ dependencies.open_zeppelin_contracts.version }}/contracts/token/ERC721/ERC721.sol#L304){target=\_blank} - an internal function from `ERC721` that safely mints new tokens. It includes checks to ensure the recipient can handle ERC721 tokens (important when minting smart contracts)
```

Comment:
```suggestion
        - [`_safeMint(to, tokenId)`]({{ dependencies.open_zeppelin_contracts.repository_url }}/blob/{{ dependencies.open_zeppelin_contracts.version }}/contracts/token/ERC721/ERC721.sol#L304){target=\_blank} - an internal function from `ERC721` that safely mints new tokens. It includes checks to ensure the recipient can handle ERC-721 tokens (important when minting smart contracts)
```

Token standards should have a hyphen in them, unless of course referring to code. Please check rest of instances and update as needed